evalai/utils/challenges.py

```diff
@@ -21,6 +21,57 @@
 requests.packages.urllib3.disable_warnings()
 
 
+def create_challenge(file, team):
+    """
+    Function to create a challenge.
+    """
+    url = "{}{}".format(get_host_url(), URLS.create_challenge.value)
+    url = url.format(team)
+
+    headers = get_request_header()
+    file = {"zip_configuration": file}
+
+    try:
+        response = requests.post(url, headers=headers, files=file)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as err:
+        if response.status_code in EVALAI_ERROR_CODES:
+            validate_token(response.json())
+            echo(
+                style(
+                    "\nError: {}\n"
+                    "\nUse `evalai challenges` to fetch the active challenges.\n"
+                    "\nUse `evalai challenge CHALLENGE phases` to fetch the "
+                    "active phases.\n".format(response.json()["error"]),
+                    fg="red",
+                    bold=True,
+                )
+            )
+        else:
+            echo(err)
+        sys.exit(1)
+    except requests.exceptions.RequestException as err:
+        echo(
```

Comment:
@KhalidRmb can we change this to a message like `invalid host team id ` if the issue is with team id otherwise the request error message

Reply:
I think this will require modification in the API, since there's no code to send an apy response for invalid host id here:     challenge_host_team = get_challenge_host_team_model(challenge_host_team_pk)